app/views/layouts/application.html.erb

```diff
@@ -1,14 +1,17 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title>Todoapp</title>
-  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
-  <%= csrf_meta_tags %>
-</head>
-<body>
-
-<%= yield %>
-
-</body>
+  <head>
```

Comment:
[MUST]
please do not remove `<%= csrf_meta_tags %>` because it protected us from XSS attack
There is many error if you remote this tag. Please note that

Reply:
i don't remove it, it for partial "_shim.html.erb"